bindings/nodejs/lib/client/client.ts

```diff
@@ -425,8 +425,8 @@ export class Client {
     /**
      * Get the token supply.
      */
-    async getTokenSupply(): Promise<u64> {
-        return (await this.getProtocolParameters()).tokenSupply;
+    async getTokenSupply(): Promise<bigint> {
```

Comment:
Why? Isn't u64 a bigint already?

Reply:
u64 is an alias for BigInt

Reply:
should we use the `u64` alias consistently? there are a few place where we don't currently.